test/tests/01-package/run.sh

```diff
@@ -156,15 +159,15 @@ docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "grep
 # kong shipped files
 docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /etc/kong/kong.conf.default"
 docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /etc/kong/kong*.logrotate"
-docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /usr/local/kong/include/kong/pluginsocket.proto"
-docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /usr/local/kong/include/wrpc/wrpc.proto"
+#docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /usr/local/kong/include/kong/pluginsocket.proto"
+#docker run ${USE_TTY} --user=root --rm ${KONG_TEST_IMAGE_NAME} /bin/sh -c "ls -l /usr/local/kong/include/wrpc/wrpc.proto"
```

Comment:
I'd prefer if this test was conditional, instead of disabled.
These tests were added because we released a version of kong that was broken without these proto files.

Reply:
I've been contemplating how to achieve that.

In the short term we could `if directory exists in kong source` but longer term I'm wondering if we need to be able to do some of these smoke tests need to live in the Kong source code repository proper (or KBT becomes part of the kong source code repositories)

Reply:
done in [313977a](https://github.com/Kong/kong-build-tools/pull/560/commits/313977a1a5361d17545e0047a93a5b87997d4e5f)